docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
+Let's say you are a data scientist working for Starbucks and you want to better understand why some stores at Brooklyn, New York, perform better than others.
 
 To begin, let's outline a workflow: 
 
-- Explore your company's data
-- Discover and enrich data thanks to the CARTO catalog
-- Analyse if the current bike stations are placed in optimal locations
+- Get and explore your company's data
+- Create the areas of influence for your stores
+- Discover and enrich your data thanks to CARTO's Data Observatory
 - And finally, share the results of your analysis with your team
 
 Let's get started!
 
-### Explore your company's data
+## Get and explore your company's data
+
+[This]() is the dataset you have to start your exploration. It contains information about the Starbucks stores. As a first exploratory step, you read it into a Jupyter Notebook using pandas.
+
+
+```python
+import pandas as pd
+
+starbucks_df = pd.read_csv('starbucks_brooklyn.csv')
+starbucks_df.head()
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>name</th>
+      <th>address</th>
+      <th>revenue</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>Franklin Ave &amp; Eastern Pkwy</td>
+      <td>341 Eastern Pkwy,Brooklyn, NY 11238</td>
+      <td>1321040.772</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>607 Brighton Beach Ave</td>
+      <td>607 Brighton Beach Avenue,Brooklyn, NY 11235</td>
+      <td>1268080.418</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>65th St &amp; 18th Ave</td>
+      <td>6423 18th Avenue,Brooklyn, NY 11204</td>
+      <td>1248133.699</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>Bay Ridge Pkwy &amp; 3rd Ave</td>
+      <td>7419 3rd Avenue,Brooklyn, NY 11209</td>
+      <td>1185702.676</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>Caesar's Bay Shopping Center</td>
+      <td>8973 Bay Parkway,Brooklyn, NY 11214</td>
+      <td>1148427.411</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+By only reading the data into a dataframe you aren't able to see at a glance where the stations are. So let's visualize it in a map!
+
+The first thing you have to do is to transform the `address` column into geometries. This process is called geocoding and CARTO provides a straightforward way to do it (you can learn more about it in the [location data services guide]()).
+
+Before being able to do it, you have to set your CARTO credentials. If you don't know your API key yet, check the [authentication guide]() to learn how to get it. In case you want to see the result of the geocoding without being logged in, here it is the [geocoded dataset]().
```

Comment:
replace "Before being able to do it, " with "In order to geocode, you have to set your CARTO credentials"